README.md

```diff
@@ -1 +1,41 @@
-# ToDo-list-app
\ No newline at end of file
+# ToDo-list App
+
+
+This is a microverse project that allows users to update tasks from a list using javascript modules
+
+Our goal here is to Build a book application with add and remove feature from scratch using JavaScripts... click [here] (https://usorfaitheloho.github.io/ToDo-list-app/) to see deployed live version
+
+![screenshot](images/screenshot-1.PNG)
+
+## Features implemented so far
+- Add book feature
+- Remove book feature
+- Saving current state of booklist to local storage 
+
+## Built With
+
+- HTML5, Javascript
+- CSS3
+- Linters
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+- Have a good knowledge on javascript objects
+- Have VSCode or other text editor installed. [Link to download VSCode](https://code.visualstudio.com/download)
+- Install node package. [Link to download node](https://nodejs.org/en/download/)
+- Have git installed.[Link to download git](https://git-scm.com/downloads)
+- Create a github repository.
+
+### Setup
+- git clone https://github.com/usorfaitheloho/ToDo-list-app.git
+- cd Awesome-booksES6/
+
```

Comment:
- [ ] Let's try to be detailed in our documentation as it can be of great benefit to users. After cloning the app, users need to install packages with `npm install`. Next, they need to run the app with `npm start` (or `npm run build`, then open `dist/index.html` file). Kindly add these commands.